Demo.java

```diff
@@ -0,0 +1,14 @@
+
+public class Demo {
+
+	public static void main(String[] args) {
+
+		House_Builder House_Builder = new Some_House_Builder();

```

Comment:
Don't use snake_case, use camelCase instead.
Vars names should start with lower case. 
Don't start names with underscores.